nodejs-java/getting-started/limitations-and-api-differences/_index.md

```diff
@@ -68,7 +68,125 @@ aspose.slides.Presentation.createPresentationFromStream(readStream, function(err
    console.log('saved to file');
 });
 ```
+### **Read Files as Byte[] in Java through node-java**
```

Comment:
We do not use node-java in our code and documentation. I propose to rename it to 'Read Files as Byte[] in Aspose.Slides for Node.js'